app/views/interactive_pages/edit.html.haml

```diff
@@ -48,6 +48,8 @@
       = link_to "Preview", preview_activity_page_path(@activity, @page), :target => 'new'
     %span.preview
       = link_to "Preview Teacher Edition", preview_activity_page_path(@activity, @page, mode: "teacher-edition"), :target => 'new'
+    %span.preview
```

Comment:
I'd recommend wrapping this in a conditional:
`if ENV['ACTIVITY_PLAYER_URL'].present?` that way we can easily enable and hide this link by un-defining the variable or setting it to the empty string.